routes.jsx

```diff
@@ -91,7 +92,7 @@ module.exports = (
       <IndexRoute component={Issues} />
       <Route path=":issue" component={Issue} />
     </Route>
-    <Route path="entry/:entryId" component={Entry} />
+    <Route path="entry/:entryId" component={Entry} onEnter={()=>{ pageSettings.setScrollPosition(); }} onLeave={()=>{ pageSettings.setRestore(); }} />
```

Comment:
`onEnter={evt =>pageSettings.setScrollPosition()}` and `onLeave={evt => pageSettings.setRestore() } />` -- as long as a full arrow function is on one line, `(...) => ...` is equivalent to `function(...) { return ... }`, so as long as the statement after the fat arrow is a single call, you don't need to wrap it in `{}` and you don't need a semi-colon at the end.



(you can use `()` instead of `evt`, but we *know* these functions come with an event handler supplied)